hack/update-codegen-crds.sh

```diff
@@ -17,24 +17,27 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-# set -o xtrace
+set -o xtrace
 
-if [[ -z "${MAKELEVEL:-}" ]]; then
-    echo 'You must invoke this script via make'
+if [[ -z "${CONTROLLER_GEN:-}" ]]; then
+    echo "You must either set CONTROLLER_GEN to the path to controller-gen or invoke via make"
```

Comment:
Since invoking by make sets that envar, these are not mutually exclusive.  They do not have to be mutually exclusive, but the reader might infer that you think they are.  How about something like "\$CONTROLLER_GEN must be defined to refer to the right binary, e.g. as is done in the Makefile"?

Reply:
changed comment as requested